lklingler-rails-tasksapp/app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,75 @@
+class TasksController < ApplicationController
+  before_action :set_task, only: [:show, :edit, :update, :destroy]
+
+  # GET /tasks
+  # GET /tasks.json
+  def index
+    @tasks = Task.order(:id)
+    @task = Task.new
+  end
+
+  # GET /tasks/1
+  # GET /tasks/1.json
+  def show
+  end
+
+  # GET /tasks/new
+  def new
+    @task = Task.new
+  end
+
+  # GET /tasks/1/edit
+  def edit
+  end
+
+  # POST /tasks
+  # POST /tasks.json
+  def create
+    @task = Task.new(task_params)
+
+    respond_to do |format|
+      if @task.save
+        format.html { redirect_to tasks_path, notice: 'Task was successfully created.' }
+        format.json { render :show, status: :created, location: @task }
+      else
+        format.html { render :new }
```

Comment:
There's no issue here but the rails generators will give you more than what you'll need. Sometimes, it's better to write your controllers from scratch.